src/twisted/conch/ssh/keys.py

```diff
@@ -624,6 +710,39 @@ def _fromDSAComponents(cls, y, p, q, g, x=None):
         return cls(keyObject)
 
 
+    @classmethod
+    def _fromECComponents(cls, x, y, curve, private_value=None):
+        """
+        Build a key from EC components.
+
+        @type x: L{int}
+        @param x: The affine x component of the public point used for verifying.
+
+        @type y: L{int}
+        @param y: The affine y component of the public point used for verifying.
+
+        @type curve: L{str}
+        @param curve: NIST name of elliptic curve.
+
+        @type private_value: L{int}
+        @param private_value: The private value.
+        """
+
+        publicNumbers = ec.EllipticCurvePublicNumbers(
+            x=x, y=y, curve=_curveTable[curve.split(b'-')[2]])
+        if private_value is None:
+            # We have public components.
+            keyObject = publicNumbers.public_key(default_backend())
+        else:
+            privateNumbers = ec.EllipticCurvePrivateNumbers(
+                private_value= private_value, public_numbers= publicNumbers)
```

Comment:
There should be no spaces after the equal signs here.  I could swear that `twistedchecker` should report this.